<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferenceDialog</class>
 <widget class="QMainWindow" name="PreferenceDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	color: #757575;
}

QListWidget::item {
	padding: 3px 0 3px 0;
}

QListWidget {
	padding-top: 16px;
	background: #f5f5f5;
}

QListWidget::item:selected {
	background: #ffffff;
	color: #757575;
}

QMainWindow {
	border: 1px solid #1cb495;
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>4</number>
    </property>
    <item row="0" column="0" rowspan="2">
     <widget class="QListWidget" name="menuListWidget">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #1cb495;
border-right: none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <property name="uniformItemSizes">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>General</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/settings/general.png</normaloff>:/icons/settings/general.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Compression</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/settings/compression.png</normaloff>:/icons/settings/compression.png</iconset>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: #1cb495;
	color: #FFFFFF;
	padding: 5px;
	border-radius: 3px;
	margin-right: 10px;
	margin-bottom: 10px;
}

QPushButton:hover {
	background-color: #1ba68a;
	color: #FFFFFF;
}</string>
        </property>
        <property name="text">
         <string>Done</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="generalPage">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <property name="verticalSpacing">
             <number>6</number>
            </property>
            <item row="7" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QComboBox" name="outputFileMethodComboBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <item>
               <property name="text">
                <string>Custom suffix</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Subfolder</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Custom folder</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="outputFileMethodLineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="placeholderText">
               <string>Leaving empty will overwrite the file</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="3">
             <widget class="QCheckBox" name="subfoldersCheckBox">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Walk through subfolders</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="browseButton">
              <property name="text">
               <string notr="true">...</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="1" colspan="2">
             <widget class="QComboBox" name="languageComboBox">
              <item>
               <property name="text">
                <string>-- System language --</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0" colspan="3">
             <widget class="QCheckBox" name="promptExitCheckBox">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Prompt before exit</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QCheckBox" name="overwriteOriginalCheckBox">
              <property name="text">
               <string>Overwrite original files</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="6" column="1" colspan="2">
             <widget class="QLabel" name="languageWarningLabel">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Language will change after restart</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="languageLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Language</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="3">
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="compressionPage">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <item row="1" column="1" colspan="2">
             <widget class="QCheckBox" name="keepCopyrightCheckBox">
              <property name="text">
               <string>Keep copyright</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>15</width>
                <height>5</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="1" colspan="2">
             <widget class="QCheckBox" name="keepDateCheckBox">
              <property name="text">
               <string>Keep date</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1" colspan="2">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>15</width>
                <height>5</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="0" colspan="3">
             <widget class="QCheckBox" name="progressiveCheckBox">
              <property name="text">
               <string>Progressive</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1" colspan="2">
             <widget class="QCheckBox" name="keepCommentsCheckBox">
              <property name="text">
               <string>Keep comments</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="3">
             <widget class="QCheckBox" name="exifCheckBox">
              <property name="text">
               <string>Keep EXIF information</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <property name="tristate">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>15</width>
                <height>5</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>overwriteOriginalCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>outputFileMethodComboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>overwriteOriginalCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>outputFileMethodLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exifCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>keepCopyrightCheckBox</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exifCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>keepDateCheckBox</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exifCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>keepCommentsCheckBox</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
